### ISSUE TITLE
[Bug]: [Accessibility]: bootstrap 5 theme - the collapseable menus for user information and private messaging system doesn't have any alternative text for screen readers

### ISSUE THREAD

**Issue by sanslash332** [e107inc/e107#5210]
### What e107 version are you using?

Latest Github version (just updated)

### Bug description

If your site are using the bootstrap 5 example theme, and your enter to it with a not signed session, you  find two sign-in related links below (or next to, i cant tell you the visual part) the navigation area, one for sign-up and sign-in.
These links are OK, and have their corresponding alternative text for screen readers.

But, after you sign-in, these links are replaced with two expandable menus. One for the private messaging system, and other for control your profile with links to sign-out, view the profile page, etc.
These expandable menus doesn't have any understandable alternative text to understand their function when you use a screen reader. They contains a simple "#" label. You can understand their function just after press it and view the expanded menu.

So is possible add a aria-label with the corresponding text that represent their function?
I've tried to find the template where these menus are built, but on the theme I just found the " {SIGNIN}" shortcode that aren't created on the theme itself, and cant found it on the built-in templates or shortcodes to check how it works and try to modify by myself.

### How to reproduce

1. grab your screen reader of preference. NVDA, JAWS or narrator on windows, voiceover on apple systems.
2. Open any web browser
3. enter to any e107 website using the default bootstrap 5 example theme. For example www.tiflojuegos.com
4. Find the sign-in and sing-up links. check that they are OK
5. enter to the sign-in option, and input your credentials to sign-in.
6. After the page loads, go to the same place where the sign-in and sign-up links were, and note that now, are two links labelled as "#"
7. Press enter on one of it, and surprise, you found that one of them can be a menu of your private messaging system, or your personal menu.

### Expected behavior

That these menus have a small name (not a complex description) with their function.
Like messages, pm, private messages or something similar for the first menu, and profile menu, profile, or simple the name of your signed user for the second.

### What browser(s) are you seeing the problem on?

Firefox, Chrome / Brave, Microsoft Edge

### PHP Version

7.4.33

**Comment by sanslash332** [comment]
O, thanks @Alex-e107nl for the tip.

I don't expected that the signing template was part of a plugin 😢
But well, lets try some changes, and if they fit wells, prepare a small PR to fix this in general. Maybe can be a small change and benefit to all blind or vision impaired people that enter to  any e107 site :-)

Thanks.

**Comment by Moc** [comment]
PR is very much appreciated!
